app/src/main/res/values-sw/strings.xml

```diff
@@ -421,9 +421,9 @@
   <string name="resume_lesson_activity_title">Rejesha Somo</string>
   <string name="resume_lesson_button">Endelea</string>
   <string name="start_over_lesson_button">Anza tena</string>
-  <string name="home_screen_good_morning_greeting_fragment">Habari ya asubuhi,</string>
-  <string name="home_screen_good_afternoon_greeting_fragment">Habari ya mchana,</string>
-  <string name="home_screen_good_evening_greeting_fragment">Habari ya jioni,</string>
+  <string name="home_screen_good_morning_greeting_fragment">Habari ya asubuhi, \u0020</string>
+  <string name="home_screen_good_afternoon_greeting_fragment">Habari ya mchana, \u0020</string>
+  <string name="home_screen_good_evening_greeting_fragment">Habari ya jioni, \u0020</string>
```

Comment:
FYI: we only need to create a string in the defaulut` strings.xml` file below  in English, and our translation service will update the language specific files. So please remove these changes and the corresponding ones from the other string files.

Reply:
Ok.